egs/thchs30/s5/run.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. ./path.sh
+
+H=`pwd`  #exp home
+n=8      #parallel jobs
+
+thchs=/nfs/public/materials/data/thchs30-openslr
+#corpus and trans directory
+#you can obtain the database by uncommting the following lines
+#( cd $thchs
+#     echo "downloading THCHS30 at $PWD ..."
+#     wget http://www.openslr.org/resources/18/data_thchs30.tgz
+#     wget http://www.openslr.org/resources/18/resource.tgz
+#     tar xvf data_thchs30.tgz && tar xvf resource.tgz 
+#)
+
```

Comment:
BTW, I just recalled I was adding two new files to the openslr. When you download only these two, are you sure you won't miss the third one? I think those were noises?